src/browser/documentation/sidebar-guides/cypher.tsx

```diff
@@ -22,73 +22,104 @@ import React from 'react'
 import ManualLink from 'browser-components/ManualLink'
 import { BuiltInGuideSidebarSlide } from '../../modules/Carousel/Slide'
 
-const title = 'Cypher'
+const title = 'Cypher Guide '
 const slides = [
-  <BuiltInGuideSidebarSlide key="first">
-    <p className="lead">Neo4j's graph query language</p>
-    <p>Neo4j's Cypher language is purpose built for working with graph data.</p>
+  <BuiltInGuideSidebarSlide key="s1">
+    <p className="lead">
+      <em>Neo4j's graph query language</em>
+    </p>
+    <p>Neo4j's Cypher language is purpose-built for working with graph data.</p>
     <ul className="big">
-      <li>uses patterns to describe graph data</li>
-      <li>familiar SQL-like clauses</li>
-      <li>declarative, describing what to find, not how to find it</li>
+      <li>Uses patterns to describe graph data.</li>
+      <li>Familiar SQL-like clauses.</li>
+      <li>Declarative, describing what to find, not how to find it.</li>
     </ul>
   </BuiltInGuideSidebarSlide>,
-  <BuiltInGuideSidebarSlide key="second">
+  <BuiltInGuideSidebarSlide key="s2">
     <h3>CREATE</h3>
-    <p className="lead">Create a node</p>
+    <p className="lead">
+      <em>Create a node</em>
+    </p>
     <p>Let's use Cypher to generate a small social graph.</p>
-    <pre className="pre-scrollable code runnable">
-      {'CREATE (ee:Person { name: "Emil", from: "Sweden", klout: 99 })'}
-    </pre>
-    <ul>
-      <li>
-        <code>CREATE</code> clause to create data
-      </li>
-      <li>
-        <code>()</code> parenthesis to indicate a node
-      </li>
-      <li>
-        <code>ee:Person</code> a variable 'ee' and label 'Person' for the new
-        node
-      </li>
-      <li>
-        <code>{'{}'}</code> brackets to add properties to the node
-      </li>
-    </ul>
+    <p>
+      <b>NOTE:</b> This guide assumes that you use an empty graph.{' '}
+    </p>
+    <ol>
+      <li>
+        Click this code block and bring it into the Editor:
+        <pre className="pre-scrollable code runnable">
+          {
+            'CREATE (ee:Person { name: "Emil", from: "Sweden", kloutScore: 99 })'
+          }
+        </pre>
+        <ul>
+          <li>
+            <code>CREATE</code> creates the node.
+          </li>
+          <li>
+            <code>()</code> indicates the node.
+          </li>
+          <li>
+            <code>ee:Person</code> – <code>ee</code> is the node variable and{' '}
```

Comment:
Question, what does the  `{' '}` do ?

Reply:
I've just checked it adds a space explicitly; see https://stackoverflow.com/questions/38547558/why-do-people-put-in-their-react-jsx. Now that I know, I am not sure if we need it in all those places. I might check all occurrences when I finish with all guides and create a separate PR to remove the unnecessary ones.